code/modules/mob/living/carbon/human/life/handle_organs.dm

```diff
@@ -11,57 +10,56 @@
 		if(!(status_flags & GODMODE)) //godmode doesn't work as intended anyway
 			reagents.metabolize(src, overdosable, L ? FALSE : TRUE)
 
-	if(!(species.species_flags & IS_SYNTHETIC))
+	if(species.species_flags & IS_SYNTHETIC)
```

Comment:
issynth()

Reply:
Is the typecheck faster than a bitflag operation?

Reply:
No, but it's the standard. The suggestion is positive.

Reply:
I agree but Life is code that is worth microopmiizing

Reply:
If you really want to optimize this, turn `handle_organs()` into a signal and there will be no need for a synth check at all.
Else it's not really worth the loss of standardization, I'd say.

Reply:
you win again